capstone/src/main/java/com/google/sps/servlets/clubs/ClubServlet.java

```diff
@@ -73,14 +72,9 @@ public void doPostHelper(
       BlobstoreService blobstore,
       DatastoreService datastore)
       throws IOException {
+    // Must use UserService to access logged in user here
```

Comment:
Can you clarify a bit why we need UserService here?

Reply:
There are a couple places I've found where the request.userPrincipal.getName() doesn't "reach," as in the email is null. So in these cases, I've just used UserService to solve the issue.